backend/api/payment_gte/server.py

```diff
@@ -0,0 +1,170 @@
+from fastapi.templating import Jinja2Templates
+from fastapi import FastAPI, Depends, HTTPException, APIRouter, Request
+
+from dotenv import load_dotenv
+from sqlalchemy.orm import Session
+
+from api.crud.base import get_db, get_company
+from api.schemas.schemas import Company
+from api.models.models import User, Customer, Subscription, Product
+
+from . import security
+
+import stripe
+import os
+
+
+load_dotenv()
+
+app = FastAPI()
+router = APIRouter()
+
+templates = Jinja2Templates(directory="api/payment_gte/templates")
+
+stripe.api_key = os.getenv("STRIPE_API_KEY")
+price_id = os.getenv("PRICE_ID")
+
+
+
+# create customer profile
+@router.post('/create-customer-object/')
+async def create_customer_object(request: Request, db: Session=Depends(get_db), tags=["Company"]):
```

Comment:
I guess the job of this function is to create a customer instance on Stripe.
If that's correct, Why are you querying all users from your db and then selecting the first one ?
Correct me if am wrong, what is going to happen is that every-time this endpoint is hit, the user at index[0] is what will be sent to stripe. Kindly confirm if this is the desired result.